conans/test/command/info_test.py

```diff
@@ -473,3 +473,58 @@ def wrong_path_parameter_test(self):
 
         self.client.run("info conanfile.txt", ignore_error=True)
         self.assertIn("ERROR: Conanfile not found", self.client.out)
+
+    def test_common_attributes(self):
+        self.client = TestClient()
+
+        conanfile = """from conans import ConanFile
+from conans.util.files import load, save
+
+class MyTest(ConanFile):
+    name = "Pkg"
+    version = "0.1"
+    settings = "build_type"
+
+"""
+
+        self.client.save({"subfolder/conanfile.py": conanfile})
+        self.client.run("export ./subfolder lasote/testing")
+
+        self.client.run("info ./subfolder")
+
+        self.assertIn("Pkg/0.1@PROJECT", self.client.user_io.out)
+        self.assertNotIn("License: MIT", self.client.user_io.out)
```

Comment:
This is a bit weird, why it would be MIT license? Don't you want to check just ``License`` instead?

Reply:
that's true, changing it now.

Reply:
Done!